test/unit/ngsiv2/commandsAmqp-test.js

```diff
@@ -235,14 +234,14 @@ describe('AMQP Transport binding: commands', function () {
             contextBrokerMock
                 .matchHeader('fiware-service', 'smartgondor')
                 .matchHeader('fiware-servicepath', '/gardens')
-                .post(
+                .patch(
```

Comment:
post is replaced by patch in this mock?

Reply:
I'm assuming this is due to changes in the iotagent-node-lib, but the test is passing for me with PATCH.